client/state/sites/selectors/get-site-woocommerce-wizard-url.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import getSiteAdminUrl from './get-site-admin-url';
```

Comment:
I'd suggest using the full path to the selector here. We tend to move those a lot, and absolute paths are much easier to update than relative ones 😅